src/containers/layout/footer/guest-footer/GuestFooter.tsx

```diff
@@ -28,6 +28,7 @@ const GuestFooter = () => {
         </Typography>
         <Typography
           component={RouterLink}
+          sx={styles.disabledLink}
```

Comment:
don`t think these styles make sense, you can remove the link or set '#'

Reply:
> don`t think these styles make sense, you can remove the link or set '#' or you can remove component link for now

you are right, the route is set to #